README.md

```diff
@@ -0,0 +1,254 @@
+![NGINX Banner](docs/img/logo.png "NGINX Banner")
+
+NGINX (pronounced "engine x" or "en-jin-eks") is the world's most popular Web Server, high performance Load Balancer, Reverse Proxy, API Gateway and Content Cache.
+
+NGINX is free and open source software, distributed under the terms of a simplified [2-clause BSD-like license](LICENSE.md).
```

Comment:
LICENSE should remain plain text IMO.

Reply:
It is now. C-style comments were also removed from the file by a previous PR